### ISSUE TITLE
Issue with output path

### ISSUE THREAD

**Issue by ashalliants** [gilmarsquinelato/serverless-spa#2]
https://github.com/gilmarsquinelato/serverless-spa/blob/3869811a83b23f5bc008de34e1146069cfbdf6ba/index.js#L479

Can we change this line to be:

```
this.webpackConfig.output.publicPath ? this.webpackConfig.output.publicPath : '/';
```

**Comment by ashalliants** [comment]
Created a pull request for you :)

**Comment by gilmarsquinelato** [comment]
Hello @ashalliants thank you for the issue and the PR, your PR was accepted and released v1.2.1.
